wordcamp.dev/provision/vvv-init.sh

```diff
@@ -62,6 +62,25 @@ else
 
 	git -C $SITE_DIR/wp-content/plugins/camptix pull origin master
 
+	wme_noroot wp plugin install supportflow            --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install polldaddy              --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install liveblog               --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-trustpay       --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-trustcard      --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-mercadopago    --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install camptix-kdcpay-gateway --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install wp-super-cache         --path=$SITE_DIR/wordpress
+	wme_noroot wp plugin install custom-content-width   --path=$SITE_DIR/wordpress
+
```

Comment:
This way they will also get added to the **.gitignore** file automatically, on [this line](https://github.com/WordPress/meta-environment/blob/master/wordcamp.dev/provision/vvv-init.sh#L50).